src/pages/partners/payment-agent/_pa-hero.tsx

```diff
@@ -43,6 +43,7 @@ const PAHero = () => {
                 external="true"
                 to="mailto:partners@deriv.com"
                 is_mail_link
+                style={{ height: '48px', 'align-items': 'center', 'padding-top': '16px' }}
```

Comment:
```suggestion
                style={{ height: '48px', alignItems: 'center', paddingTop: '16px' }}
```
style object takes the keys in camel case.

Reply:
We shouldn't set fixed button heights.

Reply:
Resolved with updating all paddings